src/reducer/productsFilterReducer.js

```diff
@@ -0,0 +1,65 @@
+export const productsFilterReducer = (state, action) => {
+  switch (action.type) {
+    case "PRICE_LOW_TO_HIGH":
+      return { ...state, sortByPrice: action.type };
+    case "PRICE_HIGH_TO_LOW":
+      return { ...state, sortByPrice: action.type };
+    case "ROLEX":
+      return {
+        ...state,
+        categories: { ...state.categories, Rolex: !state.categories.Rolex },
+      };
+    case "HUBLOT":
+      return {
+        ...state,
+        categories: { ...state.categories, Hublot: !state.categories.Hublot },
+      };
+    case "RADO":
+      return {
+        ...state,
+        categories: { ...state.categories, Rado: !state.categories.Rado },
+      };
+    case "TAGHEUER":
```

Comment:
this is fine, but you could have also done this dynamically, maybe refactor this once you have time.